docs/cis_self_assessment.md

```diff
@@ -735,9 +741,20 @@ Ensure that the admission control plugin SecurityContextDeny is set if PodSecuri
 SecurityContextDeny can be used to provide a layer of security for clusters which do not have PodSecurityPolicies enabled.
 </details>
 
-**Result:** Pass
+**Result:** **Not Scored - Operator Dependent**
```

Comment:
If a control is not scored but we pass it, put the result as pass. We do pass this one by default.